scripts/compose.yaml

```
@@ -1,4 +1,4 @@
-name: ${{values.projecSlug}}
+name: ${{values.projectSlug}}
```

```suggestion
name: ${{values.name}}
```

Thanks for catching this, if we want to pattern match on what is in the upstream this should probably be `name`

---

I think we had it right based on this file ...

[first line of yaml](https://github.com/DataBiosphere/terra-java-project-template/blob/ac5d9c8e8469640dcaa28730dabcd18be06b1933/scripts/compose.yaml#L1) here:
```yaml
name: ${{values.projectSlug}}
...
```

and [6th line of compose.yaml](https://github.com/DataBiosphere/terra-java-project-template/blob/ac5d9c8e8469640dcaa28730dabcd18be06b1933/scripts/compose.yaml#L6) here
```yaml
...
services:
  service: 
    image: ${{values.name}}
...
```

---

... I'll wait for confirmation before doing anything - 🙌

---

Ah yeah, you are right